def __init__(self, msg=''):
    RangeException.__init__(self, BAD_BOUNDARYPOINTS_ERR, msg)
